docs/source/release-notes.rst

```diff
@@ -3,17 +3,140 @@ FiftyOne Release Notes
 
 .. default-role:: code
 
+FiftyOne Teams 2.2.0
+--------------------
+*Released December 3, 2024*
+
+Includes all updates from :ref:`FiftyOne 1.1.0 <release-notes-v1.1.0>`, plus:
```

Comment:
Should there be a note about invitations and/or any other non-samples related changes?

Reply:
Ah yes, added a note for SMTP configuration 👍